hoomd/hpmc/pytest/test_compute_sdf.py

```diff
@@ -14,7 +14,7 @@
 def test_before_attaching():
     xmax = 0.02
     dx = 1e-4
-    sdf = hoomd.hpmc.compute.SDF(xmax=xmax, dx=dx)
+    sdf = hoomd.hpmc.compute.SDF(xmax=xmax, dx=dx, num_random_samples=0, beta=1)
```

Comment:
```suggestion
    sdf = hoomd.hpmc.compute.SDF(xmax=xmax, dx=dx)
```
These aren't constructor arguments?

Reply:
Correct.